site/sfguides/src/connectors_example_push_based_java/example_push_based_java_connector.md

```diff
@@ -80,16 +82,49 @@ initializes resources by creating all objects in the database needed for the def
 ## Project structure
 Duration: 3
 
+Let's take a look at the structure of this connector.
+```text
+├── Makefile
+├── example-push-based-java-connector-agent
+│    ├── build.gradle
+│    └── src
+├── example-push-based-java-connector-native-app
+│    ├── environment.yml
+│    ├── manifest.yml
+│    └── scripts
+│    │   ├── deploy.sql
+│    │   └── install.sql
+│    ├── setup.sql
+│    └── streamlit_app.py
+├── gradle
+├── gradlew
+├── gradlew.bat
+├── imgs
+├── integration-test
+├── README.md
+├── settings.gradle
+└── sf_build.sh
+```
+
 ### Native Application module
-Contains files which are needed to create a native application in Snowflake
+
+Contains files which are needed to create a Snowflake Native App:
 - `manifest.yml` - Manifest file required by the native apps model.
 - `setup.sql` - This script includes definition of all components that constitute the connector including procedures, schemas and tables.
 - `streamlit_app.py` - File which contains the UI of the connector.
-- `deploy.sql` - Script which uploads `manifest.yml`, `setup.sql` and `streamlit_app.py` to Snowflake.
-- `install.sql` - Script which creates a native application from the files uploaded by `deploy.sql` script.
+- `srcipts/deploy.sql` - Script which uploads `manifest.yml`, `setup.sql` and `streamlit_app.py` to Snowflake.
```

Comment:
Typo here, right? Should be `scripts`, not `srcipts` @sfc-gh-rzukowski

Reply:
thanks!